core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -433,6 +440,7 @@ public CHILD_CLASS globalElements(final GlobalViewElementDefinition... globalEle
             return self();
         }
 
+        @JsonSetter("globalEntities")
```

Comment:
Are these annotations needed? Usually jackson uses the public setter methods. I may be wrong.

Reply:
Removed all of the @JsonSetter methods as it was not required.